app/views/child_benefit_tax/main.html.erb

```diff
@@ -73,10 +73,17 @@
 
       <div class="question-wrapper">
         <% question_four = capture do %>
+          <p class="govuk-list">
+            You can get some of this information from your P60, P11D, employer or tax adviser. If you're not sure what income to include,
+            <a class="govuk-link" href="/government/organisations/hm-revenue-customs/contact/income-tax-enquiries-for-individuals-pensioners-and-employees">phone the Income Tax helpline</a>
```

Comment:
Just curious, why do we need a `govuk-link` class here but not in the links in _results.html.erb?

Reply:
I wondered the same and I'm not sure about the answer.
In `app/views/child_benefit_tax/_results.html.erb`, it looks ok without it:
<img width="632" alt="Screenshot 2020-10-29 at 09 32 07" src="https://user-images.githubusercontent.com/14851208/97551657-284b2180-19cb-11eb-9ec2-d4fa9bcff9e5.png">
But in `app/views/child_benefit_tax/main.html.erb`, it doesn't look ok without it:
<img width="655" alt="Screenshot 2020-10-29 at 09 31 54" src="https://user-images.githubusercontent.com/14851208/97551841-63e5eb80-19cb-11eb-93a7-a82fbdad7699.png">


My guess is some of the wrappers already apply the right class to links.